modules/ROOT/pages/traversal-framework/bidirectional_traversal_framework.adoc

```diff
@@ -0,0 +1,81 @@
+:description: The Neo4j Bidirectional Traversal Framework Java API.
+
+:org-neo4j-graphdb-bidirectional-traversal-description: {neo4j-javadocs-base-uri}/org/neo4j/graphdb/Transaction.html#bidirectionalTraversalDescription()
+
+[[Bidirectional-Traversal-Framework]]
+= The Bidirectional Traversal Framework
+
+A bidirectional traversal consists of two traversals starting from different nodes, returning the paths where both traversals collide.
+
+The Bidirectional Traversal Framework allows the description of such a traversal with the link:{org-neo4j-graphdb-bidirectional-traversal-description}[BidirectionalTraversalDescription^].
+
+[NOTE]
+====
+The Uniqueness is shared between both traversals, meaning there will be no results if the Uniqueness is set to `Uniqueness.NODE_GLOBAL`,
+as both traversals need to reach the same node to cause a collision.
+As a result of this, always set the uniqueness manually when using the `BidirectionalTraversalDescription` as the default is `Uniqueness.NODE_GLOBAL`.
```

Comment:
```suggestion
For this reason, always set the Uniqueness manually when using the `BidirectionalTraversalDescription`, since the default is `Uniqueness.NODE_GLOBAL`.
```